app/models/entities/GwasIndex.scala

```diff
@@ -76,10 +76,14 @@ object GwasIndex extends Logging {
       }
     ),
     Field(
-      "biosampleFromSourceId",
-      OptionType(StringType),
-      description = Some(""),
-      resolve = js => (js.value \ "biosampleFromSourceId").asOpt[String]
+      "biosample",
+      OptionType(biosampleImp),
+      Some("Biosample"),
+      resolve = js => {
+        val biosampleId = (js.value \ "biosampleFromSourceId").asOpt[String].getOrElse("")
```

Comment:
what should happen when id is empty? do we still get a hit? if not should we just return a None and avoid calling fetcher?